Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -8,8 +8,10 @@
 
 #include <Material/MaterialComponentController.h>
 #include <Atom/RPI.Reflect/Material/MaterialAsset.h>
+#include <Atom/RPI.Reflect/Asset/AssetUtils.h>
 #include <AzCore/Serialization/SerializeContext.h>
 #include <AtomCore/Instance/InstanceDatabase.h>
+#include <AtomLyIntegration/CommonFeatures/Mesh/MeshComponentBus.h>
```

Comment:
Note that this bus is used by Actor Component as well. (Maybe it should be called MeshProviderNotificationBus?).
Here I am only using it to address an edge case that relates to Model Assets, so it would not be applicable or required for a Decal Component implementation.